Wrappers/Python/cil/optimisation/functions/IndicatorBox.py

```diff
@@ -1,26 +1,31 @@
-# -*- coding: utf-8 -*-
-#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
-#   (Collaborative Computational Project in Tomographic Imaging), with 
-#   substantial contributions by UKRI-STFC and University of Manchester.
+# Copyright 2022 United Kingdom Research and Innovation
+# Copyright 2022 The University of Manchester
```

Comment:
```suggestion
# Copyright 2023 United Kingdom Research and Innovation
# Copyright 2023 The University of Manchester
```

Reply:
Is what you have changed the header to, the correct way to add the licensing to each file? Because most of CIL has the old header that you have deleted

Reply:
On top of not knowing what day it is, I don't know what year it is!